README.md

```diff
@@ -34,6 +34,11 @@ or:
 | `-interval`           | 10s       | How often to report stats to stdout. |
 | `-method`             | GET       | Determines which HTTP method to use when making the request. |
 | `-metric-addr`        | `<none>`  | Address to use when serving the Prometheus `/metrics` endpoint. No metrics are served if unset. Format is `host:port` or `:port`. |
+| `-use-promethus`      | `true`    | Use Promethus as metrics server |
```

Comment:
How about just a flag --metrics-server-backend, and takes in promethus or influxdb. 
We might add other backends in the future (e.g: cassandra), and we can then just use one flag to support different backends